packages/web-pkg/src/composables/actions/spaces/useSpaceActionsEditReadmeContent.ts

```diff
@@ -15,36 +15,45 @@ export const useSpaceActionsEditReadmeContent = () => {
   const userStore = useUserStore()
   const spacesStore = useSpacesStore()
   const { $gettext } = useGettext()
+  const { getDefaultMetaFolder } = useSpaceHelpers()
 
-  const handler = async ({ resources }: SpaceActionOptions) => {
-    let markdownResource = null
-    try {
-      await clientService.webdav.getFileInfo(resources[0], { path: '.space' })
-    } catch (_) {
-      await createDefaultMetaFolder(resources[0])
-      markdownResource = await clientService.webdav.putFileContents(resources[0], {
-        path: '.space/readme.md'
-      })
+  const createReadme = async (space: SpaceResource, metaFolder: Resource) => {
+    const markdownResource = await clientService.webdav.putFileContents(space, {
+      path: '.space/readme.md',
```

Comment:
We need to keep providing the path here because the resource is being used afterwards and we might need its path information. Hence this needs to wait until we make the full switch to id-based requests.

Reply:
Can you add a `FIXME`? Otherwise it might be confusing that both `path` and the tuple of `parentFolderId` and `fileName` is provided.